test/extended/networking/multinetpolicy.go

```diff
@@ -0,0 +1,102 @@
+package networking
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-network][Feature:MultiNetworkPolicy]", func() {
+	var oc *exutil.CLI
+	var ns string
+
+	oc = exutil.NewCLIWithPodSecurityLevel("multinetpol-e2e", admissionapi.LevelBaseline)
+
+	f := oc.KubeFramework()
+	podName := "multinetpol-test-pod-"
+
+	g.It("should enforce a network policies on secondary network", func() {
+		var err error
+		ns = f.Namespace.Name
+
+		g.By("creating a macvlan net-attach-def")
+		nad_yaml := exutil.FixturePath("testdata", "net-attach-defs", "macvlan-nad.yml")
+		err = oc.AsAdmin().Run("create").Args("-f", nad_yaml).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("launching pod with an annotation to use the net-attach-def")
```

Comment:
um, wait, there's already a `createNetworkAttachmentDefinition` method in `test/extending/networking/utils.go`

Reply:
Yep, saw it .But it uses dynamic client instead of Yaml files. Is it ok to turn every NAD creation (I found 4 in the code base) in a call to `createNetworkAttachmentDefinition()`? and then removing yaml files?